io.lmms.LMMS.json

```diff
@@ -0,0 +1,123 @@
+{
+  "app-id": "io.lmms.LMMS",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.12",
+  "sdk": "org.kde.Sdk",
+  "command": "lmms",
+  "rename-desktop-file": "lmms.desktop",
+  "rename-appdata-file": "lmms.appdata.xml",
```

Comment:
This is unnecessary if you install the AppData file directly.
https://github.com/hfiguiere/flathub/blob/45ea59a7439fd16e705123ff71184ad786f494d7/io.lmms.LMMS.json#L107